java/com/puppetlabs/trapperkeeper/services/webserver/jetty9/utils/InternalSslContextFactory.java

```diff
@@ -0,0 +1,164 @@
+package com.puppetlabs.trapperkeeper.services.webserver.jetty9.utils;
+
+import org.eclipse.jetty.util.log.Log;
+import org.eclipse.jetty.util.log.Logger;
+import org.eclipse.jetty.util.security.CertificateUtils;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.SSLServerSocket;
+import javax.net.ssl.SSLSocket;
+import java.io.File;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.security.cert.CRL;
+import java.util.Collection;
+
+public class InternalSslContextFactory extends SslContextFactory {
+
+    private static int maxTries = 5;
+    private static int sleepInMillisecondsBetweenTries = 1000;
```

Comment:
Sounds good. How about 100 millis and max tries 25? Then also our worst case only takes 2.5 seconds to log a failure instead of 5